lib/main-app.js

```diff
@@ -3,6 +3,7 @@ const app = electron.app
 const Menu = electron.Menu
 const ipc = electron.ipcMain
 const GhReleases = require('electron-gh-releases')
+const isDev = 'ELECTRON_IS_DEV' in process.env
```

Comment:
I'm still a little bit concern about this. Can you change to this just for safe:
```
'ELECTRON_IS_DEV' in process.env && process.env.NODE_ENV !== 'production'
```